tests/tests-tax.php

```diff
@@ -144,6 +195,25 @@ public function test_calculate_tax() {
 		$this->assertEquals( '361.58724', edd_calculate_tax( 10044.09 ) );
 	}
 
+	public function test_calculate_tax_price_includes_tax() {
+
+		// Prepare test
+		global $edd_options;
+		$origin_price_include_tax			= isset( $edd_options['prices_include_tax'] ) ? $edd_options['prices_include_tax'] : null;
+		$edd_options['prices_include_tax'] 	= 'yes';
```

Comment:
I actually wanted to use that, but there is no edd_update_option() afaik, right? (happy to add that if that makes sense to you) 

For the sake of consistency I set it via global on line #203 and re-set to original at line #214 - via global

Thoughts?